main.go

```diff
@@ -26,29 +27,45 @@ type Bot struct {
 	smsglobal  *smsglobal.Base
 	portfolio  *portfolio.Base
 	exchanges  []exchange.IBotExchange
+	db         *database.ORM
 	shutdown   chan bool
 	dryRun     bool
 	configFile string
 }
 
-const banner = `
-   ______        ______                     __        ______                  __           
+const (
+	dbname     = "gocryptotrader"
+	dbhost     = "localhost"
+	dbuser     = "gocryptotrader"
+	dbpassword = "gocryptotrader"
```

Comment:
Password in readme says should be setup to be lol123

Reply:
Will change that, cheers.